documentation/projects/proposals/additional_search_views/20230719-implementation_plan_additional_search_views.md

```diff
@@ -0,0 +1,429 @@
+the pages is directly or via links on the single results, which will all be
+captured by standard page visits. Clicking on the items will be tracked as
+`SELECT_SEARCH_RESULT` events. These events can be narrowed by pathname
+(`/search` or `/tag\*`, for example) to determine where the event occurred.
+
+Two analytics events should be added or updated:
+
+- The clicks on external creator link in the `VCollectionHeader` should be
+  tracked as `VISIT_CREATOR_LINK` events.
+
+- We should also a special event for visiting source `VISIT_SOURCE_LINK`,
+  similar to `VISIT_CREATOR_LINK`.
+
+### 9. Cleanup after the feature flag is enabled in production
+
+After the feature flag is enabled in production, we should remove the
+conditional rendering on the single result pages and remove the
+`additional_search_views` feature flag and (old) `VMediaTag` component.
+
+### Tests
+
+We should add visual-regression tests for the new views. To minimize flakiness
+due to slow loading of the images, we should probably use the
+[ `{ filter: brightness(0%); }` trick](https://github.com/WordPress/openverse/blob/b4b46b903731870015c475d2c08eebef7ec6b25b/frontend/test/playwright/visual-regression/pages/pages.spec.ts#L49-L55)
+for the images on the page.
+
+The search store tests should be updated to reflect the changes to the filters.
+
+## Dependencies
+
+### Infrastructure
+
+<!-- Describe any infrastructure that will need to be provisioned or modified. In particular, identify associated potential cost changes. -->
+
+These views potentially might cause more load on our infrastructure due to
+increase in scraping activity.
+
+### Tools & packages
+
+<!-- Describe any tools or packages which this work might be dependent on. If multiple options are available, try to list as many as are reasonable with your own recommendation. -->
+
+No new tools or packages are necessary.
+
+### Other projects or work
+
+<!-- Note any projects this plan is dependent on. -->
+
+Not applicable.
+
+## Design
+
+<!-- Note any design requirements for this plan. -->
+
+[Figma designs in the dev mode](https://www.figma.com/file/niWnCgB7K0Y4e4mgxMrnRC/Additional-search-views?node-id=22%3A13656&mode=dev)
+
+## Parallelizable streams
+
+<!-- What, if any, work within this plan can be parallelized? -->
+
+The API changes can be done independently of the frontend changes, although they
+should be finished before the final testing of the frontend changes.
+
+Adding the new components (step 3), Nuxt store update (step 4) and the
+`additional_search_views` feature flag (step 5) can be done in parallel, and are
+not dependent on anything.
+
+The work on the single result pages (step 7) can be done in parallel with the
+work on the collection pages (step 6), but should follow the previous steps.
+
+## Blockers
+
+<!-- What hard blockers exist which might prevent further work on this project? -->
+
+The main blocker could be the maintainer capacity.
```

Comment:
This is always a potential blocker, so I don't think we need to explicitly mention it.